### ISSUE TITLE
History API calls do not change the address displayed in the urlbar

### ISSUE THREAD

**Issue by JustOff** [MoonchildProductions/UXP#177]
Both `pushState()` and `replaceState()` methods of [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) do not change the address displayed in the location bar, while `window.location.href` is actually changes and new entries are added/replaced in browser history.

Here is the [demo](https://html5demos.com/history) of pushState().

PS: I do not even know where to start fixing, so I just report it.

**Comment by JustOff** [comment]
Does this mean it's better not to open individual issues related to the address bar for a while?

**Comment by mattatobin** [comment]
At this point we should because the super rapid pace is slowing and because I have to swich gears to Phoebus I can't be the omnipresent micromanager i was for 2 weeks..

tl;dr not in command anymore fallback to status quo re: issues

**Comment by JustOff** [comment]
Sorry, not sure I understood everything you said :astonished:

**Comment by wolfbeast** [comment]
It's fine to have separate issues.
"the address bar needs fixed" isn't something I can work with anyway as a statement.

**Comment by janekptacijarabaci** [comment]
This may be related (also) to:
https://github.com/MoonchildProductions/UXP/issues/121#issuecomment-381395695
`2) The Identity box is not clickable`

Maybe - I didn't look at it (yet)...

**Comment by wolfbeast** [comment]
I've had no issue with the identity panel

**Comment by wolfbeast** [comment]
Perhaps this has all been fallout from the let vs var thing? Might be a good idea to re-test if this is still the case.

**Comment by JustOff** [comment]
> 2) The Identity box is not clickable

It does not work for me too using the build from the latest trunk.

**Comment by wolfbeast** [comment]
I don't know what to say.. clicking the identity panel just brings up the normal doorhanger with site information for me.

**Comment by JustOff** [comment]
Well, now I confirm no issue with the identity box using a clean new profile. Probably something related was just not applied to the existing profile.
